### ISSUE TITLE
public window width/height

### ISSUE THREAD

**Issue by H1D** [mike-north/ember-resize#55]
Is there any reason for making window width/height [private](https://github.com/mike-north/ember-resize/blob/master/addon/services/resize.js#L11)?

Quite often I need a computed property which dependends on window size. It would be nice to do just this:

```
someProp: Em.computed( 'resizeService.windowWidth', function(){
  ...
})
```

Instead of bunch of subscribe/unsubscribe logic.
I'd like to send a PR to address this. What would be idiomatic way here?

**Comment by williamweckl** [comment]
+1

**Comment by waissbluth** [comment]
+1

**Comment by mike-north** [comment]
This was fixed in #152. Available in release `0.2.0` and later
